custom_components/frigate/icons.py

```diff
@@ -49,6 +50,8 @@ def get_icon_from_switch(switch_type: str) -> str:
         return ICON_CONTRAST
     if switch_type == "audio":
         return ICON_AUDIO
+    if switch_type == "ptz_autotracker":
```

Comment:
It's a very minor thing, but we're using both `autotrackER` and `autotrackING`. Since the Frigate config appears to use `autotrackING`, perhaps we should use that universally here?

Reply:
Frigate's backend mqtt topic uses `ptz_autotracker`, so I kept the switch name consistent with that.

Reply:
Then should we rename the variables (e.g. `ICON_AUTOTRACKER`, and all other variables in the PR?)

Reply:
It's not 100% consistent in the backend (`autotracking` is used more), and my thought process was to prepend `ptz_` to the mqtt topic to clarify the feature was for PTZ cameras. 

If consistency is important to you, I'm happy to change it to whatever you think is best!

Reply:
Thanks @hawkeye217 , makes sense. I am indeed a fan of consistency, if the backend is mixed, choose either and lets use that throughout the integration.